packages/nocodb/src/db/sql-client/lib/pg/YugabyteClient.ts

```diff
@@ -77,8 +78,12 @@ class YBClient extends PGClient {
       INNER JOIN pg_catalog.pg_class c ON (a.attrelid=c.oid)
       LEFT OUTER JOIN pg_catalog.pg_attrdef ad ON (a.attrelid=ad.adrelid AND a.attnum = ad.adnum)
       LEFT OUTER JOIN pg_catalog.pg_description dsc ON (c.oid=dsc.objoid AND a.attnum = dsc.objsubid)
-      LEFT JOIN pg_index i ON (a.attnum = any(i.indkey) and a.attrelid = i.indrelid and i.indrelid = '${args.tn}'::regclass AND i.indisprimary)
-      WHERE NOT a.attisdropped AND c.relname = '${args.tn}' and a.attnum > 0 ORDER BY a.attnum`);
+      LEFT JOIN pg_index i ON (a.attnum = any(i.indkey) and a.attrelid = i.indrelid and i.indrelid = :table::regclass AND i.indisprimary)
```

Comment:
Do we need to escape `:` here to avoid treating as identifier instead of string ? 

 `:table\\:\\:regclass`

Reply:
We don't escape on packages/nocodb/src/db/sql-client/lib/pg/PgClient.ts - columnList and seem to be working